nss-cohort-37/Bangazon-Site-Red-Jaguars
# added migration

# Description

Added migration, no further changes to seed data. Need to Install Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation NuGet package for it to compile.

Fixes # (issue)

## Type of change

# Testing Instructions

Should install the said Nuget package, and run Update-Database in your Package Manager Console.

# Checklist:
